database/factories/GenreMovieFactory.php

```diff
@@ -12,7 +12,7 @@ class GenreMovieFactory extends Factory
      *
      * @var string
      */
-    protected $model = Genre_Movie::class;
+    protected $model = Model::class;
```

Comment:
You should have a GenreMovie Model, for this pivot table 😄 

Then you can update this to `$model = GenreMovie::class`